### DISCUSSION TITLE
Vercel banned me for journalism after six years of being a customer without even talking to me first

### DISCUSSION THREAD

**Discussion by selfagency** [vercel/vercel#6923] [Help] [answered]
I have been a paying Vercel user since it was Now.sh.

A few years back, I built a project that linked data leaked from the notorious Iron March web forum, where various neo-Nazi groups, including the Atomwaffen Division, organized murder plots, to public records connecting the site's users with their real-world accountholders. The site was lauded by [journalists](https://www.ctvnews.ca/canada/map-of-hate-publication-pinpoints-canadian-users-of-neo-nazi-site-iron-march-1.4703545) and [counter-extremism](https://www.lawfareblog.com/iron-march-data-dump-provides-window-how-white-supremacists-communicate-and-recruit) researchers and was instrumental in bringing to justice numerous members of [law enforcement](https://www.usatoday.com/story/news/nation/2020/10/19/lafayette-indiana-police-fire-officer-part-neo-nazi-forum/3708374001/) and the [military](https://unicornriot.ninja/2019/neo-nazi-arms-dealer-exposed-as-canadian-navy-sailor/) in [multiple countries](https://www.bbc.com/news/uk-57806800) who were involved with murderous neo-Nazi groups. Today, Vercel decided I was violating their policy against "using leaked personal info" and banned my account. 

They didn't bother to contact me first. They didn't ask me to take down the site. They didn't take down the single site. No, they took down both my personal and business accounts, and all 30 of my sites in one fell swoop without a word, after six years of being a loyal customer.

**Comment by mcsdevv** [comment]
Hi @selfagency, I'm really sorry to hear this has happened and will work with the team immediately to understand how we got here and how we'll put this right. It certainly sounds like an error on our side, I'll investigate and be in touch via email later today.

**Reply by mcsdevv** [reply, marked as answer]
Hi @selfagency, as a quick update on this, all deployments made under the impacted accounts are live once more. I have a clearer picture of what's happened here and will continue investigating today to put together a full picture and provide you with an explanation of what didn't go as expected.

**Reply by selfagency** [reply]
@mcsdevv Thank you. I am still locked out of my account, however.

**Reply by mcsdevv** [reply]
My mistake, can you please try once more? It's now unblocked also.

**Reply by selfagency** [reply]
Thank you, I now have access again.

**Reply by selfagency** [reply]
FYI, I have removed the offending site which I will move to self-hosting.
